types/experiments.ts

```diff
@@ -39,15 +37,16 @@ export interface NimbusExperiment {
   /** The slug of the reference branch */
   referenceBranch: string | null;
   /** A specific product such as Firefox Desktop or Fenix that supports Nimbus experimentst */
-  application?: "fenix" | "firefox-desktop";
+  application: string;
+  /** Which channels should the application match? */
+  channels: Array<string>;
 }
 
 interface BucketConfig {
   /**
-   * The randomization unit. Note that client_id is not yet implemented.
-   * @default "normandy_id"
+   * The randomization unit.
    */
```

Comment:
We can possibly remove this comment since it doesn't really clarify/add anything.

Reply:
Oops, good point. I should probably revise the comments in this doc too.

Reply:
I did a quick check and don't _think_ we need to revise anything on https://mozilla.github.io/nimbus-shared/.